assets/questions/unknownAlphabet.js

```diff
@@ -0,0 +1,325 @@
+// Copyright 2017 The TIE Author, s. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Question data for Unknown Alphabet.
+ */
+
+globalData.questions['unknownAlphabet'] = {  // eslint-disable-line dot-notation
+  title: 'Unknown Alphabet',
+  starterCode: {
+    python:
+`def findAlphabet(words):
+    return ""`
+  },
+  auxiliaryCode: {
+    python:
+`class AuxiliaryCode(object):
+  @classmethod
+  def failsOnNoSolution(cls, words):
+    """
+      edges stores the edges that need to build for the alphabet graph
+      edges[ch] is a list of letters for letter ch whose ranks are bigger than
+      ch's rank.
+      degree stores the degree of each lfindAlphabetetter.
```

Comment:
findAlphabet letter? I'm not sure what this is supposed to be here. Shouldn't this also be degrees?